### DISCUSSION TITLE
PR Issues.

### DISCUSSION THREAD

**Discussion by jorgerozoh** [mlarhrouch/azure-pipeline-gpt-pr-review#42] [Q&A]
I add the task as your documentation says. and add the permissions allowing the build service agent to contribute.
![imagen](https://github.com/mlarhrouch/azure-pipeline-gpt-pr-review/assets/60438310/e9a8d878-cd9a-4e37-85af-997628c415c8)

but the task is not adding comments. I'm using OpenAI from Azure.
![imagen](https://github.com/mlarhrouch/azure-pipeline-gpt-pr-review/assets/60438310/6b1d3475-fbaa-493d-ac44-2508c72f961d)

PR-Pipeline.
![imagen](https://github.com/mlarhrouch/azure-pipeline-gpt-pr-review/assets/60438310/474b50b9-88eb-49f3-af03-a2573dfd63da)

![imagen](https://github.com/mlarhrouch/azure-pipeline-gpt-pr-review/assets/60438310/62a4c711-6960-4fe9-8630-ce97aa6555df)

What am I missing guys?

**Comment by eduards-vavere** [comment]
Same for me
